doc/user_guide/user_guide.md

```diff
@@ -290,36 +257,24 @@ severely. Available task_types are the same as the names of our available UDFs,
 
 ### 2. Model Uploader Script
-You can invoke the python script as below which allows to download the transformer 
-models from The Hugging Face hub to the local filesystem, and then from there to the BucketFS.
-
-
-#### List of options
-
-The table below lists all available options. It shows which ones are applicable for On-Prem and for SaaS backends.
-Unless stated otherwise in the comments column, the option is required for either or both backends.
-
-| Option name                  | On-Prem | SaaS | Comment                                         |
-|:-----------------------------|:-------:|:----:|:------------------------------------------------|
-| bucketfs-name                |   [x]   |      |                                                 |
-| bucketfs-host                |   [x]   |      |                                                 |
-| bucketfs-port                |   [x]   |      |                                                 |
-| bucketfs-user                |   [x]   |      |                                                 |
-| bucketfs-password            |   [x]   |      | Env. [BUCKETFS_PASSWORD]                        |
-| bucketfs-use-https           |   [x]   |      | Optional boolean, defaults to True              |
-| bucket                       |   [x]   |      |                                                 |
-| saas-url                     |         | [x]  |                                                 |
-| saas-account-id              |         | [x]  | Env. [SAAS_ACCOUNT_ID]                          |
-| saas-database-id             |         | [x]  | Optional, Env. [SAAS_DATABASE_ID]               |
-| saas-database-name           |         | [x]  | Optional, provide if the database_id is unknown |
-| saas-token                   |         | [x]  | Env. [SAAS_TOKEN]                               |
-| model-name                   |   [x]   | [x]  |                                                 |
-| path-in-bucket               |   [x]   | [x]  | Root location in the bucket for all models      |
-| sub-dir                      |   [x]   | [x]  | Sub-directory where this model should be stored |
-| task_type                    |   [x]   | [x]  | Name of the task you want to use the model for  |
-| token                        |   [x]   | [x]  | Huggingface token (needed for private models)   |
-| [no_]use-ssl-cert-validation |   [x]   | [x]  | Optional boolean, defaults to True              |
+You can invoke the Python script below which downloads the transformer 
+models from The Hugging Face hub to the local filesystem, then to the BucketFS.
+
+```shell
+python -m exasol_transformers_extension.upload_model <options>
+```
+
+For information about the available options common to all Exasol extensions please refer to the
+[documentation](https://github.com/exasol/python-extension-common/blob/0.8.0/doc/user_guide/user-guide.md) in the Exasol Python Extension Common package. 
```

Comment:
I'm not 100% sure about the versioned references to the PEC user guide.

I see the risk that in future we might forget to update these references.
Maybe it would be risky as well but still better to fall back to the latest version `PEC/doc/user_guide/user-guide.md`?

As a minimal mitigation, we could also define a variable (e.g. at the beginning of the file) and use that to create identical links in both places:
```
[pec-user-guide]: https://github.com/exasol/python-extension-common/blob/0.8.0/doc/user_guide/user-guide.md
<!-- Please check for new releases and update the reference to point to the latest version, if appropriate -->

[documentation][pec-user-guide]
```

Reply:
Referencing a particular version of PEC was precisely the intention here. This is the way to resolve conflicts when the newest version of the PEC documentation describes things that are not relevant to the latest version of the extension.
However, you've made a good point that the references are buried somewhere in the doc (and there are two of them making it worse), which makes it very easy to forget to update them. I will try your suggestion.